### PR TITLE
Feature/rule view 

### PR DESCRIPTION
**Pull request by ujjwxl**
Created a main landing page with buttons for going to game page and options for sound on and off.

This is the main landing page with buttons for single player mode and multi player mode

Screenshot (1748)

This is the options menu with sound on

Screenshot (1749)

This is the options menu with sound off

Screenshot (1750)

### CODE REVIEW

**Comment by Ramzi-Abidi** [comment]
Hey @ujjwxl thanks for the PR, I have added a home page (rules) with a checkbox to turn off /on the sound (I think it is beautiful), you just have to pull latest changes from main and rebase, you just have to adapt the home page I have created with your logic to handle the sound.
image

You can enhance it even more. And there's an issue for it.

File: src/App.tsx
Comment (Ramzi-Abidi) [inline src/App.tsx:1]: try to rebase the main branch to get the home page (I created it) and use this logic to handle the off/on sound

File: src/assets/sound-off.png
Comment (Ramzi-Abidi) [inline src/assets/sound-off.png:1]: Choose either you use these two images ( the two sound images that you've added ) to tell that the sound is on or off, and when the user click on the image you just switch between them.

or you can keep only the checkbox that I've added. 

image

Comment (ujjwxl) [reply src/assets/sound-off.png:1]: Alright, will work on it

**Review by Ramzi-Abidi — CHANGES_REQUESTED** [review]
try to enhance the home page by using the sound icons and use the logic that you implemented to set it off/on.
I will assign the issue #23 to you since you want to keep working on the home page.

**Pull request closed without merging**